specification/ilm/delete_lifecycle/DeleteLifecycleRequest.ts

```diff
@@ -36,16 +35,4 @@ export interface Request extends RequestBase {
      */
     policy: Name
   }
-  query_parameters: {
-    /**
-     * Period to wait for a connection to the master node. If no response is received before the timeout expires, the request fails and returns an error.
-     * @server_default 30s
-     */
-    master_timeout?: Duration
-    /**
-     * Period to wait for a response. If no response is received before the timeout expires, the request fails and returns an error.
-     * @server_default 30s
-     */
-    timeout?: Duration
-  }
```

Comment:
Those parameters are in fact supported: https://github.com/elastic/elasticsearch/blob/ce7a0c86fb975f76dfbea5eff1cf51602dbed184/x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/action/RestDeleteLifecycleAction.java#L38-L39